<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCalibration</class>
 <widget class="QWidget" name="FormCalibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>641</width>
     <height>191</height>
    </rect>
   </property>
   <property name="plainText">
    <string>This form allows to select input and output devices and configure it. Also it provide helps for confirure the board.
Input device:
- adjust amplifing and/or multiply factor (scale volts)
Output device:
- adjust volume - set maximal voltage which corresponds to 65535 value
The board:
- Create a guide - interactive schematic, where highlighted tested points and add some tooltips and/or hints.
Common:
- signature view - adjust resistors and see reaction

Store these settings if even input/output device is missed right now.</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>381</width>
     <height>65</height>
    </rect>
   </property>
   <property name="title">
    <string>Input audio device</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="boxAudioInputDevice"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>381</width>
     <height>65</height>
    </rect>
   </property>
   <property name="title">
    <string>Output audio device</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QComboBox" name="boxAudioOutputDevice"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>190</width>
     <height>65</height>
    </rect>
   </property>
   <property name="title">
    <string>Play tone on specific channel</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="buttonLeftOutputChannel">
      <property name="text">
       <string>Left</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonRightOutputChannel">
      <property name="text">
       <string>Right</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>151</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Check input channel</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="buttonCheckInputLevel">
      <property name="text">
       <string>Check level</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Left</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Right</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="VolumeIndicator" name="viewLeftChannelLevel"/>
      </item>
      <item row="1" column="1">
       <widget class="VolumeIndicator" name="viewRightChannelLevel"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VolumeIndicator</class>
   <extends>QGraphicsView</extends>
   <header location="global">volumeindicator.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
